src/ppx/string_util.cpp

```diff
@@ -80,9 +80,19 @@ std::string WrapText(const std::string& s, size_t width, size_t indent)
     std::string wrappedText = "";
     while (remainingString != "") {
         // Section off the next line from the remaining string, format it, and append it to wrappedText
-        size_t lineLength = remainingString.find_last_of(" \t", textWidth);
-        if (lineLength == std::string::npos) {
-            lineLength = std::min(textWidth, remainingString.length());
+        size_t lineLength = remainingString.length();
+        if (lineLength > textWidth) {
+            // Break the line at textWidth
+            lineLength = textWidth;
```

Comment:
Ah, I think the case not covered before was if it was the last line, which your if above is taking care of. You could still place the code from 83-85 inside the if statement, and I think wouldn't need the std::min. It might be simpler, up to you though.